src/deluge/gui/views/session_view.cpp

```diff
@@ -2202,11 +2203,21 @@ void SessionView::transitionToViewForClip(Clip* clip) {
 		}
 
 		else {
+
 			instrumentClipView
 			    .recalculateColours(); // Won't have happened automatically because we haven't begun the "session"
-			instrumentClipView.renderMainPads(0xFFFFFFFF, &PadLEDs::imageStore[1], &PadLEDs::occupancyMaskStore[1],
-			                                  false);
-			instrumentClipView.renderSidebar(0xFFFFFFFF, &PadLEDs::imageStore[1], &PadLEDs::occupancyMaskStore[1]);
+
+			if (((InstrumentClip*)clip)->onAutomationInstrumentClipView) {
```

Comment:
this should probably be an `else if ()` on level up (so it's on the same level as the `onKeyboardScreen` check